spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe 'Api::V1::Targets', type: :request do
+  describe 'POST #create' do
+    subject { post api_v1_targets_url, params: target_params, as: :json }
+
+    let(:topic) { create(:topic) }
+
+    context 'success' do
+      let(:target_params) do
+        {
+          'target': attributes_for(:target).merge(topic_id: topic.id)
+        }
+      end
+
+      it 'returns the target as a json' do
+        subject
+        expect(response.body).to include_json(
+          target: {
+            topic_id: topic.id,
+            title: target_params[:target][:title],
+            radius: target_params[:target][:radius],
+            longitude: target_params[:target][:longitude].to_s,
+            latitude: target_params[:target][:latitude].to_s
+          }
+        )
+      end
+
+      it 'returns a created status' do
+        subject
+        expect(response).to have_http_status(:created)
+      end
+
+      it 'creates a target' do
+        expect { subject }.to change { Target.count }.from(0).to(1)
+      end
+    end
+
+    context 'error' do
```

Comment:
I think context descriptions should have more details, like 
`context 'when the request is not valid' do`

And the same in other cases